### PR TITLE
Remove split regex args for WordPieceTokenizer

### PR DESCRIPTION
**Pull request by mattdangerw**
These have been more confusing then helpful, and we have an easy route
to do custom splitting (do it yourself before tokenization).

### CODE REVIEW

File: keras_nlp/tokenizers/word_piece_tokenizer.py [hunk 1]
```diff
@@ -112,11 +112,6 @@ class WordPieceTokenizer(tokenizer.Tokenizer):
         split: If true, input will be split according to `split_pattern`
             and `keep_pattern`. If false, input should be split before calling
             the layer.
```
Comment (jessechancy) [inline keras_nlp/tokenizers/word_piece_tokenizer.py:114]: Maybe we should update the split description to something like "split: If true, input will be split by whitespace and punctuation, and punctuations would be kept. If false, input should be split before calling the layer."?

**Review by jessechancy — APPROVED** [review]
Looks good! Just added a comment about a small docstring change.

File: keras_nlp/tokenizers/word_piece_tokenizer.py [hunk 2]
```diff
@@ -177,8 +171,6 @@ def __init__(
         lowercase: bool = True,
         strip_accents: bool = True,
```
Comment (jessechancy) [inline keras_nlp/tokenizers/word_piece_tokenizer.py:172]: Not about the change, but I think we could standardize the spacing here?

Comment (mattdangerw) [reply keras_nlp/tokenizers/word_piece_tokenizer.py:172]: You mean the spacing around the `=`? I think that's actually the way `black` does it. If you have a type annotation, spaces around `=`, if not not, no space.

**Review by chenmoneygithub — APPROVED** [review]

**Pull request merged**